todo_app/app.py

```diff
@@ -1,145 +1,185 @@
 
-app = Flask(__name__)
-app.config.from_object(Config)
-
-statusMappingList = []
-board_Name = os.environ['TRELLO_BOARD_NAME']
-board_Id = ""
-
-def build_status_mapping():
-    cboard = Board()
-    result = cboard.get_AllBoardList()
-    if result.status_code == 200:
-        for item in result.json():
-            if item['name'] == board_Name:
-                board_Id = item['id']  
-    else:        
-        print( f"Trello Board {board_Name} not found!")
-        raise SystemExit
-
-    if not (board_Id is None):
-        cboardlist = BoardList()
-        boardlists = cboardlist.get_BoardLists(id=board_Id)
-        if boardlists.status_code == 200:
-            for list in boardlists.json():
-                if list['name'] == 'To Do':
-                    statusMappingList.append(StatusMapping(id=list['id'], status='Not Started'))
-                elif list['name'] == 'Doing':
-                    statusMappingList.append(StatusMapping(id=list['id'], status='In Progress'))
-                elif list['name'] == 'Done':
-                    statusMappingList.append(StatusMapping(id=list['id'], status='Completed'))
-                else:
-                    pass  #do nothing
-        else:            
-            print("problem connecting to Trello API endpoint")
-            raise SystemExit         
-    else:
-        print( f"Trello Board {board_Name} not found!")
-        raise SystemExit
-
-def get_listId(status):
-    for item in statusMappingList:
-        if item.status == status:
-            listid = item.id
-    return listid
-
-def get_statusLabel(id):
-    for item in statusMappingList:
-        if item.id == id:
-            listStatus = item.status
-    return listStatus
-
-build_status_mapping()
-
-# error handling for 404
-@app.errorhandler(404)
-def not_found(e):
-    return render_template("error.html", error='resource not found!')
-
-@app.route('/contact')
-def contact():
-    return render_template('contact.html')
-
-# default
-@app.route('/', methods=['GET'])
-def get_index():
-    cardslist = []
-    for item in statusMappingList:
-        cbl_board = BoardList()
-        result = cbl_board.get_ListCards(item.id)    
+    statusMappingList = []
+    board_Name = os.environ['TRELLO_BOARD_NAME']
+
+
+    def build_status_mapping():
+        board_Id = ""
+        cboard = TrelloClient()
+        result = cboard.get_AllBoardList()
         if result.status_code == 200:
-            for card in result.json():
-                if not (card['due'] is None):
-                    card['due'] = (card['due']).split("T")[0]
-                cardslist.append( Card(card=card, statuslabel=item.status) )    
+            for item in result.json():
+                if item['name'] == board_Name:
+                    board_Id = item['id']  
+        else:        
+            print( f'Trello Board {board_Name} not found!' )
+            raise SystemExit
+
+        if not (board_Id is None):
```

Comment:
You set `board_Id` to the empty string `""` above, so I think this is always true

Reply:
thank you corrected.